docs/readmes/dp/debug_logs.md

```diff
@@ -7,8 +7,171 @@ id: debug_logs title: View Logs hide_title: true
+
+
+2022-04-25 15:41:22,719 DEBUG State transition from <WaitGetParametersState> to <get_obj_params>
+2022-04-25 15:41:22,720 DEBUG State transition from <GetObjectParametersState> to <wait_get_obj_params>
+2022-04-25 15:41:22,720 DEBUG Sending TR069 message: GetParameterValues {...}
+2022-04-25 15:41:22,723 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 877
+2022-04-25 15:41:22,769 DEBUG Handling TR069 message: GetParameterValuesResponse {...}
+2022-04-25 15:41:22,770 DEBUG Received object parameters: {'Device.Services.FAPService.1.CellConfig.LTE.EPC.PLMNList.1.CellReservedForOperatorUse': 'true', 'Device.Services.FAPService.1.CellConfig.LTE.EPC.PLMNList.1.Enable': 'true', 'Device.Services.FAPService.1.CellConfig.LTE.EPC.PLMNList.1.IsPrimary': 'true', 'Device.Services.FAPService.1.CellConfig.LTE.EPC.PLMNList.1.PLMNID': '00101'}
+```
+
+c) `SetParameterValues` message if a configuration change is needed to be done on the eNB side
+(the actual parameters that are set in order to enable transmission based on Domain Proxy grant
+may differ depending on the eNB model and eNB current state - please refer to your eNB device model implementation
+in `lte/gateway/python/magma/enodebd/devices`)
+
+```console
+2022-04-25 15:41:22,771 DEBUG State transition from <WaitGetObjectParametersState> to <set_params>
+2022-04-25 15:41:22,772 DEBUG Sending TR069 request to set CPE parameter values: {'SAS Radio Enable': True}
+2022-04-25 15:41:22,773 DEBUG State transition from <SetParameterValuesState> to <wait_set_params>
+2022-04-25 15:41:22,773 DEBUG Sending TR069 message: SetParameterValues {'ParameterList': {'ParameterValueStruct': [{'Name': 'Device.DeviceInfo.SAS.RadioEnable', 'Value': {'type': 'xsd:boolean', 'Data': '1'}}], 'arrayType': 'cwmp:ParameterValueStruct[1]'}}
+2022-04-25 15:41:22,777 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 723
+2022-04-25 15:41:22,837 DEBUG Handling TR069 message: SetParameterValuesResponse {'Status': '0'}
+2022-04-25 15:41:22,838 INFO Successfully configured CPE parameters!
+2022-04-25 15:41:22,839 DEBUG State transition from <WaitSetParameterValuesState> to <check_get_params>
+2022-04-25 15:41:22,840 DEBUG State transition from <GetParametersState> to <check_wait_get_params>
+2022-04-25 15:41:22,840 DEBUG Sending TR069 message: GetParameterValues {...}
+2022-04-25 15:41:22,849 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 3242
+2022-04-25 15:41:22,916 DEBUG Handling TR069 message: GetParameterValuesResponse {...}
+```
+
+d) Notify Domain Proxy state and TR069 session end
+
+```console
+2022-04-25 15:41:22,917 DEBUG State transition from <WaitGetParametersState> to <end_session>
+2022-04-25 15:41:22,917 DEBUG State transition from <BaicellsQRTBEndSessionState> to <notify_dp>
+2022-04-25 15:41:23,046 DEBUG Updating desired config based on sas grant
+2022-04-25 15:41:23,046 DEBUG Sending TR069 message: DummyInput {}
```

Comment:
If all those log statements are contained in the logs from the tar-archives, than a duplication is not necessary and you can just link line-wise instead.
This holds for all above, of course.
It is also much easier to make use of the markdown syntax for ordered lists.

Reply:
Logs trimmed down.
Used ordered list for sequencing the log snippet.

Reply:
There is duplication, but I thought it would be easier to append the docs with log snippets to provide context without forcing the user to download, untar, and browse through the unfiltered logs just to get a simple understanding of the flow.
We attach logs for a more thorough debugging reference.

Reply:
I see. Unfortunately we attach the log file as a compressed archive so line references would not work.